src/Sulu/Bundle/MediaBundle/Media/PropertiesProvider/ImagePropertiesProvider.php

```diff
@@ -23,9 +23,12 @@ class ImagePropertiesProvider implements MediaPropertiesProviderInterface
      */
     private $imagine;
 
-    public function __construct(ImagineInterface $imagine)
-    {
+    public function __construct(
+        ImagineInterface $imagine,
+        ?ImagineInterface $svgImagine = null
+    ) {
         $this->imagine = $imagine;
+        $this->svgImagine = $svgImagine;
```

Comment:
we should define this property on the class (see phpstan error )

Reply:
done!